move_base_msgs/package.xml

```diff
@@ -10,19 +12,21 @@
     <author email="contradict@gmail.com">contradict@gmail.com</author>
     <maintainer email="davidvlu@gmail.com">David V. Lu!!</maintainer>
     <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
+    <maintainer email="stevenmacenski@gmail.com">Steve Macenski</maintainer>
     <license>BSD</license>
     <url type="website">http://wiki.ros.org/move_base_msgs</url>
     <url type="bugtracker">https://github.com/ros-planning/navigation_msgs/issues</url>
 
-    <buildtool_depend>catkin</buildtool_depend>
+    <buildtool_depend>ament_cmake</buildtool_depend>
 
-    <build_depend>actionlib_msgs</build_depend>
-    <build_depend>geometry_msgs</build_depend>
-    <build_depend>message_generation</build_depend>
+    <depend>builtin_interfaces</depend>
+    <depend>rosidl_default_generators</depend>
```

Comment:
Please be more granular than `<depend>` with dependencies like `rosidl_default_generators`. Use another message package as an example:

https://github.com/ros2/demos/blob/master/pendulum_msgs/package.xml

You'll also need a `build_depend` on `rosidl_default_generators`.